testng-core/src/test/java/test/listeners/github2558/TestClassSamples.java

```diff
@@ -0,0 +1,47 @@
+package test.listeners.github2558;
+
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+public class TestClassSamples {
+
+  //  @Listeners({
```

Comment:
Wy is it commented? Document the reason or remove.

Reply:
That was not supposed to be there. Removed it.